api/tests.py

```diff
@@ -1,13 +1,154 @@
+from urllib import response
 from django.db import IntegrityError
 from django.test import TestCase
 from api.models import User, Trip
 # TestCase is a subclass of unittest, the default Python testing framework
 
 class URLTests(TestCase):
```

Comment:
great work testing these requests! i think that it might not be best practice to have everything in one file - maybe we can refactor our test suite later. but everything here is readable and makes sense to me, nice job

Reply:
definitely, I figured for now I would just get the tests written, and then it would be pretty easy to just move the tests to appropriate files